CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
```

Comment:
[Banned C function detected (strcat)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS141863.md#DS141863) 7 times
If the combination of strings is larger than the destination buffer, strcat will cbuffer overflow the destination buffer

*Fix Guidance:* Use strcat_s or strlcat if possible. strncat's size parameter excludes the null terminator, which leads to frequent mistakes


Suggestions:
[1] 
[2]